models/Daily_naive_darts/requirements.txt

```diff
@@ -0,0 +1,2 @@
+enfobench>=0.3.3

```

Comment:
Please update the version to 0.3.5, I added the helper function in this package

Reply:
I have updated the version 0.3.5. thanks,